### ISSUE TITLE
TypeScript compilation error for date function

### ISSUE THREAD

**Issue by aivarsak** [kadirahq/storybook-addon-knobs#67]
TypeScript compilation error is thrown when using this addon.
```
node_modules/@kadira/storybook-addon-knobs/storybook-addon-knobs.d.ts:26:16 
'date', which lacks return-type annotation, implicitly has an 'any' return type.
```
Adding return type "Date" or "any" resolves this issue.

**Comment by frederickfogerty** [comment]
Thanks @aivarsak, this is fixed in #69.

**Comment by aivarsak** [comment]
@frederickfogerty thanks!
